tests/test_vision_explanation.py

```diff
@@ -100,7 +118,8 @@ def test_vision_explain_loadmodel():
     _ = download_assets(modelpath)
 
     # run the main function for saliency map generation
-    res = dr.get_drise_saliency_map(imgpath, None, modelpath, 5, savepath)
+    model = (torch.load(modelpath, map_location='cpu'))
```

Comment:
Should we replace map location with `cuda if torch.cuda.is_available() else cpu` ?